frontend/src/container/Conditions.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { translate } from 'react-i18next'
+import {
+  Checkbox,
+  CUSTOM_EVENT,
+  IconButton
+} from 'tracim_frontend_lib'
+import {
+  setHeadTitle
+} from '../action-creator.sync.js'
+
+export class Conditions extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      usageConditionsCheckedList: []
+    }
+
+    document.addEventListener(CUSTOM_EVENT.APP_CUSTOM_EVENT_LISTENER, this.customEventReducer)
+  }
+
+  customEventReducer = ({ detail: { type } }) => {
+    switch (type) {
+      case CUSTOM_EVENT.ALL_APP_CHANGE_LANGUAGE:
+        this.setHeadTitle()
+        break
+    }
+  }
+
+  componentDidUpdate (prevProps, prevState) {
+    const { props } = this
+
+    if (prevProps.system.config.instance_name !== props.system.config.instance_name) {
+      this.setHeadTitle()
+    }
+  }
+
+  componentDidMount () {
+    this.setHeadTitle()
+  }
+
+  setHeadTitle = () => {
+    const { props } = this
+    props.dispatch(setHeadTitle(props.t('Conditions')))
+  }
+
+  handleClickCheckbox = (index) => {
+    const { state } = this
+    if (state.usageConditionsCheckedList.find(id => id === index)) {
```

Comment:
You shoud use the alternative of `Array.includes()` for this test:
```suggestion
    if (state.usageConditionsCheckedList.includes(index)) {
```
Because if an index === 0 is ever present in this array, this test will not work like expected.
Changing this would avoid to add +1 to the indices which are added in this array.